js/about.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable require-jsdoc */
+/* eslint-disable no-unused-vars */
+
+import {createFooter, createHeader, isSmallScreen} from './shared.js';
+
+const partnerContainer = document.querySelector('.partner');
+let isScreenCurrentlySmall;
+
+const main = () => {
+  createHeader(['home', 'program', 'join', 'sponsor']);
```

Comment:
Kindly create the top short `nav` bar section with social media links, logout, and language links aligned to the right as pointed out earlier for both homepage and about page. 👍